zabbixsender/zabbixsvc/zabbixsvc.go

```diff
@@ -66,10 +93,20 @@ func (h *JSONHandler) HandlePost(w http.ResponseWriter, r *http.Request) {
 		value = "1"
 	}
 
+	var ammountOfResolved float64
+	if req.Status == "resolved" {
```

Comment:
just do `alertsSentStats.WithLabelValues("resolved").Inc()`

else `alertsSentStats.WithLabelValues("unresolved").Inc()`

Reply:
I did it like that just because my intentions were to only increase the alertsSentStats if the alerts were successfully sent.